.eslintrc.json

```diff
@@ -28,6 +28,7 @@
     "no-param-reassign": "off",
     "react/require-default-props": "off",
     "consistent-return": "off",
+    "no-promise-executor-return": "off",
```

Comment:
this looks like a good rule to have, as it would probably catch some unintended bugs. don't think it should be turned off